### ISSUE TITLE
How sub_sample the weights?

### ISSUE THREAD

**Issue by zhoujinhai** [markshih91/mobilenet_v2_ssdlite_keras#3]
@markshih91,  I want to training the model on my dataset that have two classess, how to sub-sample the weight tensors of all the classification layers?

**Comment by DubiousCactus** [comment]
The original writer of this SSD implementation has a very nice tutorial for that: [https://github.com/pierluigiferrari/ssd_keras/blob/master/weight_sampling_tutorial.ipynb](https://github.com/pierluigiferrari/ssd_keras/blob/master/weight_sampling_tutorial.ipynb)

**Comment by zhoujinhai** [comment]
yes, I have tried it, and change the layers name from ```conv4_3_norm_mbox_conf``` to  ```ssd_cls1conv2```,  but  it  show that ```bias:0``` does not exist after running.

**Comment by DubiousCactus** [comment]
Well I can't help you, I haven't tried it myself...

**Comment by zhoujinhai** [comment]
Ok Thanks


发自我的iPhone

------------------ Original ------------------

**Comment by liguilan1227** [comment]
> yes, I have tried it, and change the layers name from `conv4_3_norm_mbox_conf` to `ssd_cls1conv2`, but it show that `bias:0` does not exist after running.

I met the problem,too. Have you solve it? Please help me.
